### PR TITLE
Pelion Documentation

### PR DESCRIPTION
**Pull request by ZakariasL**

### CODE REVIEW

File: docs/pelion.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+kube-router documentation [General Requirements](https://www.kube-router.io/docs/user-guide/#requirements)
+
+Make sure ipset is installed
+```
+apt update && apt install -y ipset
+```
+
+## Build binary without docker
+```
+BUILD_IN_DOCKER=false make kube-router
```
Comment (udayakumar17) [inline docs/pelion.md:37]: Should this be the default method to build kube-router? Shouldn't we also add the method to build binary with docker?

File: docs/pelion.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Pelion
+
+## Requirements
+
+kube-router documentation [General Requirements](https://www.kube-router.io/docs/user-guide/#requirements)
+
+Make sure ipset is installed
+```
+apt update && apt install -y ipset
```
Comment (udayakumar17) [inline docs/pelion.md:13]: Isn't docker a pre-requisite for build with docker? Also in build without docker method don't we need to install Go?

**Review by udayakumar17 — CHANGES_REQUESTED** [review]

File: docs/pelion.md (hunk 1 above)
Comment (ZakariasL) [reply docs/pelion.md:37]: Added more build information and with/without docker alternatives.

File: docs/pelion.md (hunk 2 above)
Comment (ZakariasL) [reply docs/pelion.md:13]: Updated the documentation to contain the requirements needed for building.

**Review by mohitsingalarm — APPROVED** [review]

**Review by singhbhoopesh — APPROVED** [review]
LGTM

**Pull request merged**